templates/partials/question.html

```diff
@@ -58,12 +58,16 @@
 {% set question_guidance %}
   {%- if question.guidance -%}
     {% set contents = question.guidance.contents %}
-    {% call onsPanel({
-      "id": "question-guidance-" ~ question.id,
-      "classes": "ons-u-mb-m"
-    }) %}
-      {% include 'partials/contents.html' %}
-    {% endcall %}
+    {% for item in contents %}
```

Comment:
I think this has broken the `test_question_guidance.json` schema, its rendering the block 4 times now, I don't think the loop will work here, might need to be in contents?

Reply:
since this got through, might need to update a functional test to cover this?

Reply:
Yep just found this issue too

Reply:
Added a fix, just need to add a test